api/src/resource-extraction.js

```diff
@@ -23,7 +23,34 @@ const isAttachmentExtractable = name => {
 // Map of attachmentName -> attachmentDigest used to avoid extraction of unchanged documents
 let extractedDigests = {};
 
-const createFolderIfDne = x => !fs.existsSync(x) && fs.mkdirSync(x);
+const createFolderIfDne = folderPath => !fs.existsSync(folderPath) && fs.mkdirSync(folderPath);
+
+const removeDirectoryRecursive = (dirPath) => {
+  if (!fs.existsSync(dirPath)) {
+    return;
+  }
+
+  const files = fs.readdirSync(dirPath) || [];
+
+  files.forEach(fileName => {
+    const filePath = `${dirPath}/${fileName}`;
```

Comment:
I think it's safer to use `path.join(dirPath, fileName)` to construct paths rather than concatenating strings manually.

Reply:
👍 fixed